qiskit/transpiler/oracle_compiler/utils.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Internal utils for Oracle Compiler"""
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.library.standard_gates import ZGate, TGate, SGate, TdgGate, SdgGate, U1Gate, \
+    XGate, HGate, U3Gate
+from .exceptions import OracleCompilerError
+
+
+def tweedledum2qiskit(tweedledum_circuit, name=None, qregs=None):
+    """ Converts a Tweedledum circuit into a Qiskit circuit. A Tweedledum circuit is a
+    dictionary with the following shape:
+        {
+        "num_qubits": 2,
+        "gates": [{
+            "gate": "X",
+            "qubits": [1],
+            "control_qubits": [0],
+            "control_state": "1"
+        }]
+    Args:
+        tweedledum_circuit (dict): Tweedledum circuit.
+        name (str): NName for the resulting Qiskit circuit.
+        qregs (list(QuantumRegister)): Optional. List of QuantumRegisters on which the
+           circuit would operate. If not provided, it will create a flat register.
+
+    Returns:
+        QuantumCircuit: A Qiskit quantum circuit.
```

Comment:
Maybe it's obvious but it's a bit nicer to read in the documentation 🙂 
```suggestion
        QuantumCircuit: The Tweedledum circuit converted to a Qiskit circuit.
```